src/Analyzer.PowerShellRuleEngine/PowerShellRuleEngine.cs

```diff
@@ -115,13 +118,16 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(TemplateContext templateContext)
                     },
                     Execution = new ExecutionOption
                     {
-                        NotProcessedWarning = false,
```

Comment:
Was this removed intentionally? I don't fully remember why it was set but I think there was a reason

Reply:
No, this option was renamed/deprecated. See the attribute "UnprocessedObject = ExecutionActionPreference.Ignore".